.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,36 @@
+name: Pre-commit checks
+
+on:
+  push:
+    branches-ignore:
+      - 'generated'
+      - 'codegen/**'
+      - 'integrated/**'
+      - 'stl-preview-head/**'
+      - 'stl-preview-base/**'
+  pull_request:
+    branches-ignore:
+      - 'stl-preview-head/**'
+      - 'stl-preview-base/**'
+
+jobs:
+  pre-commit:
+    timeout-minutes: 10
+    name: Pre-commit hooks
+    runs-on: ${{ github.repository == 'stainless-sdks/llama-stack-client-go' && 'depot-ubuntu-24.04' || 'ubuntu-latest' }}
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.fork
```

Comment:
this `if` is failing I think for the PR since you are not running in a fork. I don't think you need this conditional at all